extensions/hibernate-orm/deployment/src/main/java/io/quarkus/hibernate/orm/deployment/HibernateOrmConfig.java

```diff
@@ -117,27 +151,29 @@ public boolean isAnyPropertySet() {
     public static class HibernateOrmConfigDatabase {
 
         /**
-         * Control how schema generation is happening in Hibernate ORM.
-         * <p>
-         * Same as JPA's javax.persistence.schema-generation.database.action.
+         * Select whether the database schema is generated or not.
+         *
+         * `drop-and-create` is awesome in development mode.
+         *
+         * Accepted values: `none`, `create`, `drop-and-create`, `drop`, `update`. The default is `none`.
```

Comment:
Same here, this custom tag makes it impossible to extract the allowed values in the table, so probably should use an enum, no?